main.py

```diff
@@ -73,9 +35,5 @@ def convert_website_to_stamp(_website, maximum_pages):
 
     url, maximum_pages = get_user_input()
 
-    _website = Website(url)
-
-    if not _website.is_valid:
-        LOGGER.error("Invalid URL!")
-    else:
-        convert_website_to_stamp(_website, maximum_pages)
+    _stampifier = Stampifier(url, maximum_pages)
+    _stamp_file_path = _stampifier.stampify()
```

Comment:
If you throw errors in stampify(), you can catch it here and log it. I guess it is better to throw exception instead of returning empty string for invalid outputs.

Reply:
Updated accordingly. Let me know if further changes are required!